### ISSUE TITLE
Android DTO partition is missing AVB signature

### ISSUE THREAD

**Issue by armijnhemel** [kaitai-io/kaitai_struct_formats#449]
Nitpicking, when looking at the diagram here:

https://source.android.com/devices/architecture/dto/partitions

it seems that the DTO partition should also include the AVB signature. Currently this is not the case. The current .ksy describes the dtbo.img file but that is only part of a DTO partition. I can see two solutions:

1. add AVB signature to the .ksy
2. rename the .ksy to indicate it only describes the dtbo.img file

**Comment by generalmimon** [comment]
Good catch! The [`android_dto`](https://github.com/kaitai-io/kaitai_struct_formats/blob/2a1ddf0/archive/android_dto.ksy) spec was merged just a few days ago from a pull request by @evdenis (https://github.com/kaitai-io/kaitai_struct_formats/pull/415), so I hope he might have an idea how to handle this.

However, the test files that @evdenis provided in https://github.com/kaitai-io/kaitai_struct_formats/pull/415#issue-570642168 apparently do not have any AVB signature at the end of file either - the files are parsed till the end (and not with `size-eos: true`).
